sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/StorageApiWriteUnshardedRecords.java

```diff
@@ -263,11 +281,12 @@ void invalidateWriteStream() {
             // thread has already invalidated
             // and recreated the stream).
             @Nullable
-            StreamAppendClient cachedAppendClient = APPEND_CLIENTS.getIfPresent(streamName);
+            StreamAppendClient cachedAppendClient =
+                APPEND_CLIENTS.getIfPresent(getStreamAppendClientCacheEntryName());
```

Comment:
nit - store cachedEntryName in a local variable instead of recomputing twice here

Reply:
done